style/index.css

```diff
@@ -0,0 +1,79 @@
+/* index.css */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 60%;
+  font-family: Arial, sans-serif;
+  padding: 20px;
+  align-items: center;
+}
+
+#form {
+  width: 60%;
+  margin: auto;
+}
+
+h1 {
+  text-align: center;
+}
+
+#bookList {
+  margin-top: 20px;
+}
+
+.showBookListBorder {
+  border: 1px solid black;
+}
+
+.book {
+  display: flex;
+  justify-content: space-between;
+  border: 1px solid #ccc;
+  padding: 10px;
+  margin: 10px;
+}
+
+.add-book {
+  padding: 30px 0 10px;
+  text-align: center;
+  font-size: 20px;
+  font-weight: 800;
+}
+
+input[type="text"] {
+  width: 100%;
+  padding: 5px;
+  margin-bottom: 10px;
+}
+
+#submit {
+  display: block;
+  width: 30%;
+  padding: 10px;
+  background-color: #4caf50;
+  color: #fff;
+  border: none;
+  cursor: pointer;
+  margin-left: auto;
+  border-radius: 5px;
+}
+
+#submit:hover {
+  background-color: #45a049;
+}
+
+button {
+  padding: 5px 10px;
+  background-color: #f44336;
+  color: #fff;
+  border: none;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: #e53935;
+}
```

Comment:
You did well in implementing some of the requested changes by the previous reviewer .As stated by the previous reviewer,please the border you added should be for the entire book container and not just the area where the books are displayed,

Your design :

![image](https://github.com/Obote/AwesomeBooks/assets/20537712/e019dfb8-0cdf-481f-89ef-a176766f83a7)

Template :
![image](https://github.com/Obote/AwesomeBooks/assets/20537712/f660ab74-0b25-466c-b237-70ef7f9b3eac)

Reply:
Thank you for your review @usorfaitheloho , the border has been added.
![awesomebooks](https://github.com/Obote/AwesomeBooks/assets/111312788/16e7f2dc-d0fb-451e-a115-0b9b46eb6315)